cci_nepal/analysis/wp2_data_scoping/database_sample/database_sample_descriptive.py

```diff
@@ -0,0 +1,190 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import cci_nepal
+import nltk
+import re
+import logging
+
+# %%
+# nltk.download("words")
+
+# %%
+project_directory = cci_nepal.PROJECT_DIR
+logging.info(project_directory)
+
+# %%
+db = pd.read_excel(
+    f"{project_directory}/inputs/data/wp2_data_scoping/PDM_ Datasheet.xlsx"
+)
+
+# %%
+db.shape
+
+# %%
+db.drop(db.columns[2], axis=1, inplace=True)
+
+
+# %%
+def clean_df_columns(df):
+    df.columns = df.columns.str.replace("[^a-zA-Z0-9\-\s]+", "", regex=True)
+    df.columns = df.columns.str.lstrip()
+
+
+# %%
+clean_df_columns(db)
+
+# %%
+db = db.replace("[^a-zA-Z0-9\-\s]+", "", regex=True)
+
+# %%
+db.head(5)
+
+# %%
+percent_missing = db.isnull().sum() * 100 / len(db)
+missing_value_df = pd.DataFrame(
+    {"column_name": db.columns, "percent_missing": percent_missing}
+)
+missing_value_df.sort_values("percent_missing", inplace=True)
+
+# %%
+missing_value_df["percent_missing"].plot(kind="bar")
+plt.xticks([])
+
+# %%
+plt.figure(figsize=(15, 10))
+missing_value_df["percent_missing"].tail(60).plot(kind="bar")
+
+# %%
+# db.to_excel('clean-temp.xlsx', index=False)
```

Comment:
Saving the cleaned data to outputs/inputs directory to follow the structure layout may be preferable?